docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -100,16 +107,33 @@ and reject the whole document.
 
 |=======================================================================
 
+
+[[indexing-approach]]
+[float]
+==== Indexing approach
+GeoShape types are indexed by decomposing the shape into a triangular mesh and
+indexing each triangle as a 7 dimension point in a BKD tree. This provides
+near perfect spatial resolution (down to 1e-6 decimal degree precision) since all
```

Comment:
Isn't it `1e-7`? (`360 / 2^32` ~= `8.4e-8` ~= `1e-7`)

Reply:
:+1:  That's right... I was wrong.